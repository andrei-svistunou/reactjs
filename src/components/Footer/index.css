.main-footer {
  background-color: #444;
  padding: 15px 25px;
}