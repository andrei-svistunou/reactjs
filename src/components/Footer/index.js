import React from 'react';
import Logo from '../Logo';
import './index.css';

const Footer = () => (
  <div className='main-footer'>
    <Logo />
  </div>
);

export default Footer;
