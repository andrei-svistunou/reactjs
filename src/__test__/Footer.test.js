import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../components/Footer';

describe('<Footer />', () => {
  test('should render <Footer />', () => {
    const footer = shallow(<Footer />);

    expect(footer).toMatchSnapshot();
  });
});
