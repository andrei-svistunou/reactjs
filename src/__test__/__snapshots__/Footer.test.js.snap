// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render <Footer /> 1`] = `
<div
  className="main-footer"
>
  <Logo
    text="netflixroulett"
  />
</div>
`;
